server/src/main/scala/za/co/absa/atum/server/api/http/Routes.scala

```diff
@@ -95,14 +95,14 @@ trait Routes extends Endpoints with ServerOptions {
   private def createSwaggerRoutes: HttpRoutes[HttpEnv.F] = {
     val endpoints = List(
       createCheckpointEndpointV1,
-      postCheckpointEndpointV2,
+//      postCheckpointEndpointV2,
       createPartitioningEndpointV1,
-      postPartitioningEndpointV2,
+//      postPartitioningEndpointV2,
       patchPartitioningAdditionalDataEndpointV2,
```

Comment:
Do you guys know, if we use this endpoint in Agent now? @salamonpavel @lsulak

Reply:
I don't think so, PATCH wasn't introduced into the agent.

Agent is using this endpoint: `POST /api/v2/writeAdditionalData`

Reply:
actually perhaps we should adjust the agent to work with this PATCH because otherwise we got incompatibility in 0.3.0 between agent and server